### PR TITLE
Show filter type being used

### PR DESCRIPTION
**Pull request by mike-eason**
Hello there,

Instead of having to type `.abc.def` you can **ignore the starting period entirely** and just type `abc.def`.

### CODE REVIEW

**Comment by mike-eason** [comment]
I also added icons for filter types which resolves issue #6, however I couldn't find a decent svg for jq so I just used the node.js logo.

**Comment by wellsjo** [comment]
@mike-eason awesome, thanks! I'll review this today.

File: app/js/Editor.js [hunk 1]
```diff
@@ -122,6 +123,10 @@ class Editor {
       this.hideRightPanel()
       return
     }
+
+    if (filter.substring(0, 1) !== '.')
```
Comment (wellsjo) [inline app/js/Editor.js]: This excludes a lot of `jq` commands such as `keys`, `map()`, `del()`, etc.

https://stedolan.github.io/jq/manual/#Basicfilters

File: app/assets/question.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<?xml version="1.0" ?><!DOCTYPE svg  PUBLIC '-//W3C//DTD SVG 1.1//EN'  'http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd'><svg enable-background="new 0 0 128 128" height="128px" id="Layer_1" version="1.1" viewBox="0 0 128 128" width="128px" xml:space="preserve" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><path d="M64.049,114.172v-2l-0.006,2c-27.642,0-50.129-22.486-50.129-50.128c0-27.642,22.487-50.13,50.129-50.13  s50.129,22.488,50.129,50.13C114.172,91.686,91.687,114.172,64.049,114.172z M64.043,17.914c-25.436,0-46.129,20.694-46.129,46.13  c0,25.436,20.693,46.128,46.129,46.128h0.006c25.433,0,46.123-20.692,46.123-46.128C110.172,38.608,89.479,17.914,64.043,17.914z"/><g><path d="M64.043,74.354c-1.104,0-2-0.896-2-2v-9.756c0-1.104,0.896-2,2-2c5.27,0,9.557-4.286,9.557-9.556   c0-5.271-4.287-9.56-9.557-9.56s-9.557,4.289-9.557,9.56c0,1.104-0.896,2-2,2s-2-0.896-2-2c0-7.477,6.081-13.56,13.557-13.56   S77.6,43.565,77.6,51.042c0,6.796-5.026,12.439-11.557,13.409v7.903C66.043,73.458,65.147,74.354,64.043,74.354z"/><circle cx="64.042" cy="85.098" r="3.796"/></g></svg>
```
Comment (wellsjo) [inline app/assets/question.svg]: I think an empty hexagon would look nice instead...I'll look around

File: index.html [hunk 3]
```diff
@@ -65,7 +65,8 @@
 
   <!-- Bottom bar / filter input -->
   <div class="bottom-wrapper hidden">
-    <img src="app/assets/shuffle.svg" class="filter-icon">
+    <img id="filterIcon" src="app/assets/question.svg" class="filter-icon">
```
Comment (wellsjo) [inline index.html]: let's use classes only, since tabs will be implemented soon (and there will be multiple filter-icons)

**Comment by wellsjo** [comment]
@mike-eason for the `jq` logo, could we use their actual logo instead of the nodejs one? How did you make those svgs? `https://stedolan.github.io/jq/jq.png`

**Comment by mike-eason** [comment]
@wellsjo I just used an online [png to svg converter](http://image.online-convert.com/convert-to-svg), quite nifty really.

Anyway, I added the jq logo for you.

**Comment by wellsjo** [comment]
@mike-eason awesome nice job! hope you don't mind I changed pr title to make more sense. I just did a pretty big refactor which should affect some of the code here. I see there are some conflicts now

**Comment by mike-eason** [comment]
No problemo!

If it's too much of a hassle, just close this PR and manually add what you need.

Cheers.

**Comment by wellsjo** [comment]
I merged our work together https://github.com/wellsjo/JSON-Splora/pull/16

**Pull request closed without merging**